fmn/rules/utils.py

```diff
@@ -198,6 +244,58 @@ def _get_pkgdb2_packages_for(config, username, flags):
     return dict(packages)
 
 
+def _get_pagure_packages_for(config, username, flags):
+    """
+    Get the packages a user is associated with from pagure.
+
+    Args:
+        config (dict): The application configuration.
+        username (str): The FAS username to fetch the packages for.
+        flags (list): The type of relationship the user should have to the
+            package (e.g. "watch", "point of contact", etc.).
+    """
+    log.debug("Requesting pagure packages for user %r" % username)
+
+    # XXX - We don't yet have a way to query for all projects that a user
+    # *watches*.  See https://pagure.io/pagure/issue/2421
+    valid_flags = ['point of contact', 'co-maintained']#, 'watch']
+
+    bogus = set(flags) - set(valid_flags)
+    if bogus:
+        log.error("%r are not valid owner flags for %r." % (bogus, username))
+        # ... but, proceed.
+
+    valid = set(flags) & set(valid_flags)
+    if not valid:
+        log.error("No valid owner flags by which to query.")
+        return dict()
+
+    default = 'https://src.fedoraproject.org/pagure/api'
```

Comment:
Since this is never referenced after its single use directly below, you could probably just use it directly instead of assigning it to a variable